Formula/tinyscheme.rb

```diff
@@ -20,35 +20,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -\?", 1)
+  end
 end
 
 __END__
---- a/makefile  2011-01-16 20:51:17.000000000 +1300
-+++ b/makefile  2012-04-08 22:38:11.000000000 +1200
+--- a/makefile
++++ b/makefile
```

Comment:
I think the same could be said about just about any use of `inreplace`.

My concern is that this patch is basically just un-commenting the little OS X section but was understandably characterized by @DomT4 as "fairly hefty to carry inline". It's also difficult to see what parts of the patch are the uncommenting of what's already there and what's parts are actually a substantive change and why.

Based on https://gist.github.com/anonymous/5520f144682f37711d95bf3a7cb271d3 I'm tempted to suggest we not even use the makefile and run the commands inline so it's transparent what this actually does, which in its entirety is

```
gcc -fpic -pedantic -I. -c -g -Wno-char-subscripts -O  -DUSE_DL=1 -DUSE_MATH=1 -DUSE_ASCII_NAMES=0 -DOSX -DInitFile="\"/usr/local/Cellar/tinyscheme/1.41/share/init.scm"\"  scheme.c
gcc -fpic -pedantic -I. -c -g -Wno-char-subscripts -O  -DUSE_DL=1 -DUSE_MATH=1 -DUSE_ASCII_NAMES=0 -DOSX -DInitFile="\"/usr/local/Cellar/tinyscheme/1.41/share/init.scm"\"  dynload.c
gcc -shared -o libtinyscheme.dylib scheme.o dynload.o -ldl -lm
ar crs libtinyscheme.a scheme.o dynload.o
gcc -fpic -pedantic -o scheme -g -Wno-char-subscripts -O scheme.o dynload.o -ldl -lm
```